decidim-core/app/forms/decidim/amendable/review_form.rb

```diff
@@ -13,7 +13,7 @@ class ReviewForm < Decidim::Amendable::Form
       attribute :body, String
       attribute :user_group_id, Integer
 
-      validates :id, :amendable_gid, :emendation_gid, presence: true
+      validates :id, :amendable_gid, :title, :body, :emendation_gid, presence: true
```

Comment:
`amendable_fields` (in this case `:title` and `:body`) should be dynamic as amendable is configurable.

https://github.com/CodiTramuntana/decidim/blob/feature/amendments-proposals-on-proposals/decidim-core/lib/decidim/amendable.rb#L26